Sources/BraintreePayPal/BTPayPalClient.swift

```diff
@@ -463,6 +469,8 @@ import BraintreeDataCollector
         approvalURL = appSwitchURL
         webSessionReturned = false
         
+        webAuthenticationSession.prefersEphemeralWebBrowserSession = experiment == "InAppBrowserNoPopup"
```

Comment:
Are we capturing this in our analytics somewhere on the PayPal side? Just want to make sure we are able to track if this is making a difference or not in terms of conversion. I am sure this was already discussed so am just catching up. :)